mlrun/projects/project.py

```diff
@@ -2417,6 +2418,8 @@ def set_model_monitoring_function(
         :param application_class:       Name or an Instance of a class that implements the monitoring application.
         :param application_kwargs:      Additional keyword arguments to be passed to the
                                         monitoring application's constructor.
+        :param local_path:              Path to a local directory to save the downloaded monitoring-app code files in,
+                                        in case func is a hub URL.
```

Comment:
Add to the docstring of the `func` param that it can begin with `hub://` and explain what it does.